### PR TITLE
Modified /pom.xml for Maven Central.#298

### PR DESCRIPTION
**Pull request by bturamatuyu**
Please review #298 .

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -35,25 +35,16 @@
...
-                <enabled>true</enabled>
-            </releases>
-            <snapshots>
-                <enabled>false</enabled>
-            </snapshots>
-            <id>terasoluna-gfw-releases</id>
-            <url>http://repo.terasoluna.org/nexus/content/repositories/terasoluna-gfw-releases/</url>
-        </repository>
-        <repository>
+    <repository>
```
Comment (kazuki43zoo) [inline pom.xml]: - [ ] indent is incorrect.

Comment (bturamatuyu) [inline pom.xml]: @kazuki43zoo san
Modified indent in /pom.xml.

Comment (kazuki43zoo) [inline pom.xml]: @bturamatuyu 

Your fix is incorrect. Please check again.

Comment (bturamatuyu) [inline pom.xml]: @kazuki43zoo san
sorry.
I corrected the indent correctly.

**Pull request merged**